tests/unit/abilities-api/wpAbilitiesRegistry.php

```
@@ -281,6 +282,21 @@ public function test_register_invalid_meta_type() {
 		$this->assertNull( $result );
 	}
 
+	/**
+	 * Should reject ability registration with invalid show in REST type.
+	 *
+	 * @covers WP_Abilities_Registry::register
+	 * @covers WP_Ability::prepare_properties
+	 *
+	 * @expectedIncorrectUsage WP_Abilities_Registry::register
+	 */
+	public function test_register_invalid_show_in_rest_type() {
+		self::$test_ability_args['show_in_rest'] = 5;
+
+		$result = $this->registry->register( self::$test_ability_name, self::$test_ability_args );
+		$this->assertNull( $result );
```

Looks like this should throw an `InvalidArgumentException`, right? Not just be null?

---

I think we catch it at this level, but it should throw in `WP_Ability` constructor. I will cover it accordingly, as we apparently don't have that part covered for other validated properties 👍🏻

---

Addressed in[9f21064](https://github.com/WordPress/abilities-api/pull/103/commits/9f21064ca08f2ce50553f8fea8b0acd6948bd61c).